### PR TITLE
Created Calculator using python GUI

### LINKED ISSUE #188: Calculator using GUI (PYTHON)
Hey, Greeting's of the day !!
(A clear and concise description of what the feature is.)
**I want to add Calculator to your repository made using GUI (Python)** i.e ..     Project-Guidance/Web Development/Basic/Calulator 
It can do basic arithmetic operation .     

Yeas i read all the contributing Guidelines ..
I would like to work on this under **GSSOC'22**
Please assign this to me .
Thank's regard .. :crossed_fingers: 
(Write your answer here.)

### PR DESCRIPTION
**Pull request by Vikash-8090-Yadav**
Hey , Hop u r doing good !!  :heart: 
I created the calculator using python gui . its a basic calculator which used for the basic airthmrtic operation's .Along with this i 

also make the readme file for the calculator 
Issue no: 
Fixes #188 

@Kushal997-das  plss chek this

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.

### CODE REVIEW

File: Web Development/Basic/Calculator_GUI/Readme.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+
+![Screenshot from 2022-03-03 20-19-18](https://user-images.githubusercontent.com/85225156/156588847-c61fa546-6365-48d1-98eb-12af07038423.png)
+
+
+<br><br>
+<h4>When the button is pressed for the airthmatic operation the display view is :</h4>
+
+![Screenshot from 2022-03-03 19-57-32](https://user-images.githubusercontent.com/85225156/156588187-bb8f1184-11cd-4d60-a388-573fc7a0b61c.png)
```
Comment (Kushal997-das) [inline Web Development/Basic/Calculator_GUI/Readme.md:11]: Please reduce the size of image !!

File: Web Development/Basic/Calculator_GUI/Readme.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<hr>
+<h4>When the button is pressed for the Result especially "=" button :</h4>
+
+![Screenshot from 2022-03-03 19-57-40](https://user-images.githubusercontent.com/85225156/156587954-828b1ba9-99af-42fa-bee4-0b0cbf0a19d5.png)
```
Comment (Kushal997-das) [inline Web Development/Basic/Calculator_GUI/Readme.md:15]: Reduce the size of the image !

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Vikash-8090-Yadav Please reduce the image size which ever exits in your `readme.md` file

**Review by Kushal997-das — APPROVED** [review]
@Vikash-8090-Yadav Great 👍 
LGTM 👍 

Please check @photon149

File: Web Development/Basic/Calculator_GUI/Calculator_GUI.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
+from tkinter import *
+
+# class Tk:
+
+
+root = Tk()
+root.geometry("744x900")
+root.title("GROUP- 7 PROJECT - CALCULATOR CU (LAB MST ) A.I == PYTHON GUI")
```
Comment (photon149) [inline Web Development/Basic/Calculator_GUI/Calculator_GUI.py:8]: @Vikash-8090-Yadav Please Update the title of the project.

**Review by photon149 — CHANGES_REQUESTED** [review]
Please do the requested changes

**Review by photon149 — APPROVED** [review]
looks good to me.
@prakhar3062 please have a look.

**Comment by Kushal997-das** [comment]
@prakhar3062 Please have a look !!

**Review by prakhar3062 — APPROVED** [review]
@Kushal997-das  everything is fine

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Vikash-8090-Yadav Do one changes this project uses only Tkinter in python . So this should go under **Desktop application** [Under this ](https://github.com/Kushal997-das/Project-Guidance/tree/main/Desktop%20Application/Basic/Python)

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Your dasktop image is visible so crop the image only calculator should shown in the readme file.

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Pull request merged**